config/initializers/riiif_initializer.rb

```diff
@@ -13,15 +13,18 @@
 # each image. If you are using hydra-file_characterization, you have the height & width
 # cached in Solr. The following block directs the info_service to return those values:
 Riiif::Image.info_service = lambda do |id, _file|
+  Rails.logger.debug("[RIIIF] H/W CHECK FOR #{id}")
```

Comment:
@sethaj I can see some value in the logging statement below, but this one will run for every load of a Leaflet map and doesn't seem very informative?

Reply:
I'll remove it. It was just there to see how often it was actually called and was hitting solr.

Reply:
Ah, debug so not in production <facepalm>. I'm just going to merge this!